### PR TITLE
Hotfix: About-US Hero with gutters

### PR DESCRIPTION
**Pull request by sebworks**
Hotfix: About-Us Hero with gutters

## Changes
- Updated  ' static/css/bureau.less' to add gutters and white background. 

## Notes
- Adjusted image position for beautification purposes.

## Test
- Navigate to 'http://localhost:7000/the-bureau/ '.

## Review 
- @jimmynotjim  
- @anselmbradford 
- @schaferjh   
- @duelj 
- @KimberlyMunoz 

## Screenshots

Desktop
screen shot 2015-05-20 at 1 15 13 pm

### CODE REVIEW

File: static/css/bureau.less [hunk 1]
```diff
@@ -182,14 +182,26 @@
...
         });
     }
+
+    .youtube-player_image {
+        // Optical adjustment for beautification purposes.
+        top: -1 * @gutter__em;
+        width: @grid_wrapper-width;
+
+        .respond-to-max(825px, {
+            // Adujustment for YouTube image height/weight ratio.
```
Comment (KimberlyMunoz) [inline static/css/bureau.less]: Tiny typo. `Adjustment.`

File: static/css/bureau.less [hunk 2]
```diff
@@ -253,21 +266,17 @@
 }
 
 .youtube-player_image {
```
Comment (KimberlyMunoz) [inline static/css/bureau.less]: Can we group the positioning and display properties separately? It makes it easier to find where different things are later, especially in something like this with a lot of css properties.

https://github.com/cfpb/front-end/blob/master/css.md#property-order

**Comment by KimberlyMunoz** [comment]
I tested this on a giant screen and it looks great. A few minor comments above.

image

File: static/css/bureau.less [hunk 3]
```diff
@@ -216,6 +228,7 @@
 }
 
 .youtube-player_container {
+    background: #fff;
```
Comment (anselmbradford) [inline static/css/bureau.less]: Could change to `background: @white`

**Comment by jimmynotjim** [comment]
Working well locally and working great. :+1: after making the suggested changes above

**Pull request merged**